sympy/core/evalf.py

```diff
@@ -1439,7 +1439,14 @@ def evalf(self, n=15, subs=None, maxn=100, chop=False, strict=False, quad=None,
 
         chop : bool or number, optional
             Specifies how to replace tiny real or imaginary parts in
-            subresults by exact zeros.
+            subresults by exact zeros. e.g.
+            >>> from sympy import Rational, exp
+            >>> a = Rational('37751345442791/1000000000000000000000000')
+            >>> b = exp(-24)
+            >>> (a-b).evalf(n=20, chop=True)  
```

Comment:
@Jatin2020-24 Github Actions are failing due to `trailing white spaces` in this line.